packages/nimble-components/src/label-provider/core/index.ts

```diff
@@ -65,6 +69,12 @@ export class LabelProviderCore
     @attr({ attribute: 'loading' })
     public loading: string | undefined;
 
+    @attr({ attribute: 'scroll-backward' })
+    public scrollBackward: string | undefined;
+
+    @attr({ attribute: 'scroll-forward' })
+    public scrollForward: string | undefined;
```

Comment:
Reminder, the label public api changes require angular and blazor updates. Can do in follow-on PR or in this one.

Reply:
If  done in this PR I guess reset me. Maybe better to do in a follow-up